src/main/java/com/bakdata/common_kafka_streams/KafkaStreamsApplication.java

```diff
@@ -248,15 +252,18 @@ protected void runCleanUp() {
         this.inputTopics.stream()
                 .filter(topic -> !topic.isBlank())
                 .forEach(topic -> runResetter(topic, this.brokers, this.getUniqueAppId()));
+
         if (this.deleteOutputTopic) {
             if (!this.outputTopic.isBlank()) {
                 this.deleteTopic(this.outputTopic);
+                this.resetSchemaRegistry();
```

Comment:
this needs to be done for the error topic as well. Please move it to the deleteTopic() method